runtime-modules/content/src/benchmarks/benchmarking.rs

```diff
@@ -3249,6 +3249,154 @@ benchmarks! {
+
+    // WORST CASE SCENARIO
+    // STATE COMPLEXITY
+    // - curator owned channel
+    // - channel-owning curator group has max number of permissions per level
+    // - curator number is max
+    // - curator has max number of agent permissions
+    // - channel has max size:
+    //   - all feature paused (except necessary ones for extr to succeed)
+    //   - max channel assets
+    //   - max collaborators
+    // INPUT COMPLEXITY
+    // - remark message byte-length: b
+    channel_owner_remark {
+        let b in 1 .. MAX_BYTES_METADATA;
+        let (channel_id, group_id, lead_account_id, curator_id, curator_account_id) =
+            setup_worst_case_scenario_curator_channel_all_max::<T>(false)?;
+        let msg = vec![1u8].repeat(b as usize);
+        let origin = RawOrigin::Signed(lead_account_id);
+    }: _(origin, channel_id, msg.clone())
+        verify {
+            assert_last_event::<T>(
+                <T as Config>::Event::from(
+                    Event::<T>::ChannelOwnerRemarked(
+                        channel_id,
+                        msg
+                    )
+                ).into()
+            );
+        }
+
+    // WORST CASE SCENARIO
+    // STATE COMPLEXITY
+    // - curator owned channel
+    // - channel-owning curator group has max number of permissions per level
+    // - curator number is max
+    // - curator has max number of agent permissions
+    // - channel has max size:
+    //   - all feature paused (except necessary ones for extr to succeed)
+    //   - max channel assets
+    //   - max collaborators
+    // INPUT COMPLEXITY
+    // - remark message byte-length: b
+    channel_agent_remark {
+        let b in 1 .. MAX_BYTES_METADATA;
+        let (channel_id, group_id, lead_account_id, curator_id, curator_account_id) =
+            setup_worst_case_scenario_curator_channel_all_max::<T>(false)?;
+        let origin = RawOrigin::Signed(curator_account_id);
+        let actor = ContentActor::Curator(group_id, curator_id);
+        let msg = vec![1u8].repeat(b as usize);
+    }: _(origin, actor, channel_id, msg.clone())
+        verify {
+            assert_last_event::<T>(
+                <T as Config>::Event::from(
+                    Event::<T>::ChannelAgentRemarked(
+                        actor,
+                        channel_id,
+                        msg,
+                    )
+                ).into()
+            );
+        }
+
+    // WORST CASE SCENARIO
+    // STATE COMPLEXITY
+    // - curator owned channel
+    // - channel-owning curator group has max number of permissions per level
+    // - curator number is max
+    // - curator has max number of agent permissions
+    // - channel has max size:
+    //   - all feature paused (except necessary ones for extr to succeed)
+    //   - max channel assets
+    //   - max collaborators
+    // - video has max size
+    //   - max video assets
```

Comment:
Worst case scenario should include nft being in english auction state with max. possible number of whitelisted members (in order to bloat the size of the video).
Also please add a mention that the nft should be owned by the channel.